fractions/swagger/src/main/resources/modules/io/swagger/main/module.xml

```diff
@@ -5,14 +5,14 @@
     <artifact name="io.swagger:swagger-models:${version.io.swagger}"/>
     <artifact name="io.swagger:swagger-annotations:${version.io.swagger}"/>
 
-    <artifact name="com.google.guava:guava:${version.swagger.guava}"/>
     <artifact name="org.reflections:reflections:${version.swagger.reflections}"/>
-    <artifact name="org.javassist:javassist:${version.swagger.javassist}"/>
+    <artifact name="org.javassist:javassist:${version.org.javassist}"/>
     <artifact name="org.apache.commons:commons-lang3:${version.swagger.commons.lang}"/>
 
   </resources>
 
   <dependencies>
+    <module name="com.google.guava"/>
```

Comment:
I figured since there is a general desire to reduce the overall deliverable size, I could help in a small way.

Reply:
org.javassist should be picked up by its module and not the artifact directly, the same as you've done for guava